devices/ikea.js

```diff
@@ -245,6 +245,18 @@ const ikea = {
                 }
             },
         },
+        command_dots: {
+            cluster: 64639,
+            type: 'raw',
+            convert: (model, msg, publish, options, meta) => {
+                const lookup = {
+                    1: 'dot_single',
+                    2: 'dot_double',
+                    3: 'dot_long',
```

Comment:
Does it make sense to have the `dot_` prefix? (e.g. `single` looks better imo)

Reply:
I've been discussing this with a colleague for quite a bit. The device has two extra buttons, one with 1 dot and one with 2 dots. (You can see it on the image below the prev/next buttons).
So this becomes `1dot_single`, `2dot_long` and so on. if we remove the dot, it would only be `1_single`, and so on.
While its not perfect, I feel the dot version is easier to understand.

That said, it's your project and I can live with both.